producers/src/unit-test/java/uk/gov/moj/sdt/producers/sdtws/SdtInternalEndpointPortTypeTest.java

```diff
@@ -119,6 +160,13 @@ private UpdateRequestType createUpdateRequest() {
 
         final HeaderType header = new HeaderType();
         header.setSdtRequestId("12345678");
+        final UpdateStatusType statusType = new UpdateStatusType();
+        statusType.setCode(UpdateStatusCodeType.ACCEPTED);
+        final ErrorType errorType = new ErrorType();
+        errorType.setDescription("MOCK_ERROR");
+        errorType.setCode("MOCK_CODE");
+        statusType.setError(errorType);
+        request.setStatus(statusType);
```

Comment:
Has this been added to ensure coverage? If so it should be checked in a test - at the moment it doesn't appear to be checked in any of them. It also may not be a realistic set of data - can you have a request with a status of OK but also with an error?

Same comment applies to createUpdateResponse().